public/src/components/PostItem/CommentComponent.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+    faAngleUp,
+    faAngleDown,
+    faCommentAlt,
+} from '@fortawesome/free-solid-svg-icons';
+import methods from '../../utils/methods';
+
+const {
+    getDate,
+    reformatTextToHtml,
+} = methods;
+
+export default class CommentsComponent extends React.Component {
+    constructor(props) {
+        super(props);
+        this.calculateDate = this.calculateDate.bind(this);
+        this.getCommentClassName = this.getCommentClassName.bind(this);
+        this.getOffsetLines = this.getOffsetLines.bind(this);
+    }
+
+    getCommentMessageHtml() {
+        const { body } = this.props;
+        return (
+            <div className="comment-body__text"
+                 dangerouslySetInnerHTML={{ __html: reformatTextToHtml(body) }}
+            />
+        );
+    }
+
+    getOffsetLines() {
+        const { depth } = this.props;
+        const elementsToReturn = [];
+        for (let i = 0; i < depth; i += 1) {
+            const element = <div key={i} className="offsetLine" />;
+            elementsToReturn.push(element);
+        }
+        return elementsToReturn;
+    }
+
+    getCommentClassName() {
+        const { depth } = this.props;
+        return depth === 'null' || depth === '0'  ? 'parent' : 'child';
+    }
+
+    calculateDate() {
+        const { created } = this.props;
+        return created && created !== 'null' ? getDate(created) : null;
+    }
+
+    render() {
+        const { author, score } = this.props;
+        return (
+            <div className={`comment ${this.getCommentClassName()}`}>
```

Comment:
This approach for creating classes is fine, but you can also try `classnames` library if you'd like

Reply:
fixed https://github.com/YuriyBC/Reddit-archive/commit/55b77abd89f4da9f77e78904d4a58a19ee920e17